Tasks/VsTest/VSTest.ps1

```diff
@@ -37,133 +37,141 @@ import-module "Microsoft.TeamFoundation.DistributedTask.Task.TestResults"
+                    }
+                    Publish-TestResults -Context $distributedTaskContext -TestResultsFiles $resultFiles -TestRunner "VSTest" -Platform $platform -Configuration $configuration -RunTitle $testRunTitle
+                }
             }
             else
             {
-                if(!$publishResultsOption)
+                if($testRunTitle)
+                {
+                    Write-Warning (Get-LocalizedString -Key "Update the agent to try out the '{0}' feature." -ArgumentList "custom run title")
+                }
+            
+                if($publishRunLevelAttachmentsExists)		
                 {
-                    Write-Warning (Get-LocalizedString -Key "Update the agent to try out the '{0}' feature." -ArgumentList "opt in/out of publishing test run attachments")
+                    Publish-TestResults -Context $distributedTaskContext -TestResultsFiles $resultFiles -TestRunner "VSTest" -Platform $platform -Configuration $configuration -PublishRunLevelAttachments $publishResultsOption
                 }
-                Publish-TestResults -Context $distributedTaskContext -TestResultsFiles $resultFiles -TestRunner "VSTest" -Platform $platform -Configuration $configuration -RunTitle $testRunTitle
+                else
+                {
+                    if(!$publishResultsOption)
+                    {
+                        Write-Warning (Get-LocalizedString -Key "Update the agent to try out the '{0}' feature." -ArgumentList "opt in/out of publishing test run attachments")
+                    }
+                    Publish-TestResults -Context $distributedTaskContext -TestResultsFiles $resultFiles -TestRunner "VSTest" -Platform $platform -Configuration $configuration
+                }		
             }
         }
         else
         {
-            if($testRunTitle)
-            {
-                Write-Warning (Get-LocalizedString -Key "Update the agent to try out the '{0}' feature." -ArgumentList "custom run title")
-            }
-            
-            if($publishRunLevelAttachmentsExists)		
-            {
-                Publish-TestResults -Context $distributedTaskContext -TestResultsFiles $resultFiles -TestRunner "VSTest" -Platform $platform -Configuration $configuration -PublishRunLevelAttachments $publishResultsOption
-            }
-            else
-            {
-                if(!$publishResultsOption)
-                {
-                    Write-Warning (Get-LocalizedString -Key "Update the agent to try out the '{0}' feature." -ArgumentList "opt in/out of publishing test run attachments")
-                }
-                Publish-TestResults -Context $distributedTaskContext -TestResultsFiles $resultFiles -TestRunner "VSTest" -Platform $platform -Configuration $configuration
-            }		
+            Write-Host "##vso[task.logissue type=warning;code=002003;]"
+            Write-Warning (Get-LocalizedString -Key "No results found to publish.")
```

Comment:
Can we modify this warning to provide:  $testResultsDirectory & specify that it will search for *.trx